web/package.json

```diff
@@ -7,7 +7,7 @@
     "storybook": "start-storybook --ci -p 9001 -c .storybook",
     "build-storybook": "build-storybook -c .storybook -o .storybook-dist/public",
     "make-state-icons": "babel-node scripts/stateIcons.js",
-    "scaffold": "mkdir -p dist && cp src/index.html dist/ && cp ../Staticfile dist/ && cp -r src/static dist/",
+    "scaffold": "rm -rf dist && mkdir dist && cp ../Staticfile dist/ && cp -r src/static dist/",
```

Comment:
Deletes the `dist` directory and starts over every time.  Otherwise those JS files with hashes in their filenames would accumulate in here forever.  In deployment it doesn't matter because we're starting with a clean filesystem every time anyway, but for building locally, it's nice to start with a clean slate.